locale/de/manageiq.po

```diff
@@ -26827,6 +26833,9 @@ msgstr "Ungültige Authentifizierungsnachweise"
 msgid "Invalid button action"
 msgstr "Ungültige Schaltflächenaktion"
 
+msgid "Invalid button action!"
+msgstr "Ungültige Schaltflächenaktion!"
```

Comment:
This string was removed, so is this just from an old run?

Reply:
yes